test/C/src/ActionDelay.lf

```diff
@@ -1,28 +1,29 @@
 // Test logical action with delay.
-target C;
+target C
```

Comment:
I recall that we wanted to use semicolons in the C target for consistency, but I'm not sure.

Reply:
@cmnrd explained why his preference was _not_ to use the semicolons, and I agreed with him. I'm not sure I remember exactly what he said, but reasons include
- It makes sense to distinguish between LF code and target code because the two are semantically different. This makes the situation quite unlike the expressions discussion where we wanted to make an LF tuple expression look like a Python tuple. Unlike those target-code-like expressions, the lines of LF code that we would terminate with semicolons do not mean the same, and do not try to mean the same, as the statements appearing inside target code blocks. Therefore, those LF expressions should not try to _look_ the same as target code.
- The semicolons are unnecessary. They don't encode semantic meaning. They don't provide informative visual cues to programmers. They don't provide useful information to the compiler (except possibly in the case of _bugs_ that should be _fixed_, maybe in the case of the IPv4 and IPv6 rules that pollute our grammar if I remember correctly). Therefore, the principle of parsimony in all things written forbids the use of semicolons.

Insertion of the semicolons in the formatter would also increase the number of special cases in our implementation.

Reply:
The higher-level comment here is that it isn't a serious problem to have semicolons, and we can insert them if we like, but because there is a cost to having them, there should be a reason for using them. The hard-headedness of C programmers who do not want to see any new syntax probably should not be the only reason.

Reply:
This sounds reasonable to me. I'm personally fine with not having them...

Reply:
Given that no one else has followed up on this, is this PR ready to merge?

Reply:
Yes, let's do it. We should follow up on the required semicolon in the `at`clause and revisit that construct all together. It doesn't appear in the tests and is not very well documented, so I don't think we need to address it before merging this in.